lib/endpoints/servers.js

```diff
@@ -1754,6 +1949,19 @@ function attachTo(http, app) {
         }),
         Server.factoryReset);
 
+    // Update recovery configuration on the server
+    http.post({
+        path: '/servers/:server_uuid/recovery-config',
+        name: 'ServerRecoveryConfig' },
+        ensure({
+            connectionTimeoutSeconds: 60 * 60,
```

Comment:
Does 60 minutes seem too long?

Reply:
It's actually the default for everything there so I decided to do not modify all of them.